imageregistry/v1/types_imagepruner.go

```diff
@@ -12,6 +12,7 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ImagePrunerList is a slice of ImagePruner objects.
+// +openshift:compatibility-gen:level=1
```

Comment:
level 4, exposed=false and make a note to revisit with the image team as a followup to figure  out if users touch this.